hw/ip/aes/data/aes_testplan.hjson

```diff
@@ -132,5 +124,119 @@
       milestone: V2
       tests: []
     }
+    {
+      name: regwen
+      desc: '''
+            Make sure that writes to  ctrl_aux_shadowed does not take effect if regwen is disabled. '''
+      milestone: V2
+      tests: []
+    }
```

Comment:
this can be verified by csr test since HW won't update it and RAL can predict how it affects the other CSRs. Coverage can be automatically collected as well. See #10634.

perhaps we can remove this one as well as the covergroup.

Reply:
good plan, thank you @weicaiyang weicai

Reply:
fixed